### PR TITLE
implement add sources - cable modeling

### PR DESCRIPTION
**Pull request by gmalinve**
clock and pwl sources implementation

### CODE REVIEW

**Review by MaxJPRey — APPROVED** [review]
LGTM.

File: pyaedt/modules/CableModeling.py [hunk 1]
```diff
@@ -374,284 +393,597 @@ def add_cable_to_bundle(self):
...
+                msg = "AddPwlSource and UpdatePwlSource fields can't have the same value."
+                raise ValueError(msg)
+            try:
+                # Check if user action is to remove the source
+                if values["SourcesToRemove"]:
+                    if values["SourcesToRemove"] in self.existing_sources_names:
+                        self.source_to_remove = values["SourcesToRemove"]
                     else:
-                        msg = 'The only accepted wire standard is "ISO".'
+                        msg = "Cable to remove doesn't exists in the current design."
```
Comment (MaxJPRey) [inline pyaedt/modules/CableModeling.py:874]:
```suggestion
                        msg = "Cable to remove doesn't exist in the current design."
```

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM

**Comment by Samuelopez-ansys** [comment]
@MaxJPRey  @maxcapodi78 I fixed the IPython Unittest, but I don't know why codevoc is failing, please could you check?

@gmalinve

**Pull request merged**